docs/checking-broken-links.md

```diff
@@ -72,9 +82,85 @@ yarn linkcheck http://localhost:8080/server/index.html | grep "BROKEN"
 Note: in the example output above, only the first entry is a genuine broken link.
 All the others in the list are example links, and therefore not broken.
 
-### A general linkchecker utility
+## Public Available Linkcheckers
+
+### Linkcheck by Filip Hracek
+
+This is a extreme fast and very comfortable / configurable link checker.
+
+A description of ``linkcheck`` can be found [here](https://github.com/filiph/linkcheck#linkcheck).
+Follow [this](https://github.com/filiph/linkcheck#step-1-install-dart) procedure,
+to install ``linkcheck``, which needs ``dart``:
+
+Because ``linkcheck`` provides the possibility to use a file to exclude search patterns,
+it is a good advice to create a file with following predefined content. In this example,
+the file is named ``my_skip_file.txt`` and saved one level below the local docs repository.
+Adopt the content to your needs.
+
+```console
+# exclude check to sub-pages of owncloud
+https://owncloud.org/news
+https://owncloud.org/support
+https://owncloud.org/install
+
+# do not crawl branches or client repositories
+http://localhost:8080/server/10.0
+http://localhost:8080/server/10.1
+http://localhost:8080/server/10.2
+http://localhost:8080/branded_clients
+http://localhost:8080/desktop
+http://localhost:8080/android
+http://localhost:8080/ios
+
+# exclude because of denied by robots.txt anyway
+https://github.com
+https://gist.github.com
+https://www.samba.org
+https://linux.die.net
+https://mycloud.org
+https://www.google.de
+https://www.tscp.org
+
+```
+
+As a good advice, first start to check excluding external pages / sites.
```

Comment:
"As a good advice," - this could be removed.
Otherwise it could say something like:
"It is good practice to first start by checking while excluding external pages / sites"